index.html

```diff
@@ -814,150 +814,156 @@ <h2>Privacy Considerations</h2>
+      </pre>
+
+      <p>
+Finally, we combine the two hashes, use the private key with the combined hash to
+compute the Ed25519 signature, and then base58-btc encode the signature.
+      </p>
+
+      <pre class="example" title="Combine hashes of Proof Options and Credential (hex)">
+565a2884ebb2d38aa34871108074ab51631ec812d33eb2473178bce19937ad096c6b2795e7fa33a9fb28062527142b3c6edf7ba239942439b6f0bb0851b3cce3
+      </pre>
+
+      <pre class="example" title="Signature of Combined Hashes (hex)">
+473fb02a4aaf5863a2ef33f104bd55617e40907bc311e29e87278d15d7596f201639f41ec0e00db11159e9139f673d9257558e1f0134e1f67ac73f91ed89670b
+      </pre>
+
+      <pre class="example" title="Signature of Combined Hashes base58-btc">
+z2RczMj342tVhAjgjEPV4TeHbi2ggnTRKTc5BFQCgaWJ3nhcg5HgCeC2eV4Lc1fYdhfoLyPjxoq4BtqrsyNvxZ8nE
+      </pre>
+
+      <p>Assemble the signed credential with the following two steps:</p>
+      <ol>
+        <li>
+Add the <code>proofValue</code> field with the previously computed base58-btc 
+value to the proof options document.
+        </li>
+        <li>
+Set the <code>proof</code> field of the credential to the augmented proof 
+option document.
+        </li>
+      </ol>
+
+      <pre class="example" title="Signed Credential">
 {
-  "vc_template_0": {
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://www.w3.org/2018/credentials/examples/v1",
-      "https://w3id.org/security/suites/ed25519-2020/v1",
-    ],
-    "id": "http://example.gov/credentials/3732",
-    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-    "issuer": "https://example.com/issuer/123",
-    "issuanceDate": "2020-03-10T04:24:12.164Z",
-    "credentialSubject": {
-      "id": "did:example:456",
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Science and Arts"
-      }
-    }
-  },
-  "vc_0": {
-    "@context": [
-      "https://www.w3.org/2018/credentials/v1",
-      "https://www.w3.org/2018/credentials/examples/v1",
-      "https://w3id.org/security/suites/ed25519-2020/v1"
-    ],
-    "id": "http://example.gov/credentials/3732",
-    "type": ["VerifiableCredential", "UniversityDegreeCredential"],
-    "issuer": "https://example.com/issuer/123",
-    "issuanceDate": "2020-03-10T04:24:12.164Z",
-    "credentialSubject": {
-      "id": "did:example:456",
-      "degree": {
-        "type": "BachelorDegree",
-        "name": "Bachelor of Science and Arts"
-      }
-    },
-    "proof": {
-      "type": "Ed25519Signature2020",
-      "created": "2019-12-11T03:50:55Z",
-      "proofValue": "z5SpZtDGGz5a89PJbQT2sgbRUiyyAGhhgjcf86aJHfYcfvPjxn6vej5na6kUzmw1jMAR9PJU9mowshQFFdGmDN14D",
-      "proofPurpose": "assertionMethod",
-      "verificationMethod": "https://example.com/issuer/123#key-0"
+  "@context": [
+    "https://www.w3.org/2018/credentials/v1",
+    "https://www.w3.org/2018/credentials/examples/v1",
+    "https://w3id.org/security/suites/ed25519-2020/v1"
```

Comment:
Not for this PR, but a future PR should update this to use the v2 context (the example will look much cleaner, IMHO).

Reply:
Will do. I'm currently working on test vectors for the "DataIntegrityProof" representation so will make sure to use the latest context!